doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
+- Change VERSION variable in build-server
+- Commit and push the changes
+
+== Check ==
+
+- Everything blue on https://build.libelektra.org/
+- Trigger release pipeline with dry run
+- Inspect the logs, artifacts and the website from the pipeline
+  - [ ] error logs (*.error) must be empty
+  - [ ] check in the (test-*) folder if all tests passed
+  - [ ] check if version is correct in the version file
+  - [ ] 
+
+== Preperation ==
+- Finalize release notes (hash sums etc are already known)
+- cd ~e && cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
+
+== Publish ==
+- Trigger release pipeline without dry run
+- (Possible) rebuilds from website for last changes in the release notes
+
+= Announce =
+send out release mails&post on social media #676
+(Personal notice: see also NOTIFY)
+
+
+= TODO =
+- Docker container with sudo rights so that the built deb-packages can be installed and tested in the release pipeline
+- Move ~e/scripts/dev/update-infos-status into release pipeline
+- Move updating of alpine linux docker image into release pipeline
+- "Increment Version Number" could be automized by using parameterized build jobs
+- Add ABI Tests
+- Include external test suites into pipeline
+
+
+----------------------------------------------------------------------
+
+This describes what to do for a manual release:
```

Comment:
Not needed anymore, everyone will be happy if this does not need to be done anymore :100: